recipes/reads2graph/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -ex
+
+# Add the Conda environment's bin directory to the PATH
+export PATH="${PREFIX}/bin:${PATH}"
+
+#Set the CXX environment variable to the specific g++ version
+export CXX=$(which g++)
+export CC=$(which gcc)
```

Comment:
These two lines are not needed

Reply:
Yes, these two lines are indeed not needed. I used this to help me debug. I will remove it. Thanks, Martin.